sonar-xml-plugin/pom.xml

```diff
@@ -170,8 +170,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>2550000</maxsize>
-                  <minsize>2500000</minsize>
+                  <maxsize>2680000</maxsize>
+                  <minsize>2630000</minsize>
```

Comment:
Do we know what's causing the size increase here?

Reply:
Since the only "real" change is the increment of the org.sonarsource.parent version number: I think the new parent version includes additional dependencies.

Do you think it's worth investigating this further?